codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -151,38 +151,39 @@ update these file paths accordingly.
 
   /// The entrance point of examining and code signing an engine artifact.
   Future<void> validateAll() async {
+    for (String passwordFilePath in [
+      codesignAppstoreIDFilePath,
+      codesignTeamIDFilePath,
+      appSpecificPasswordFilePath,
+    ]) {
+      await readPassword(passwordFilePath);
```

Comment:
Is the output of this method supposed to be saved somewhere?  I'd suggest to refactor readPassword to return the value from the file and remove the validations. If the user is passing a file with wrong values that is the responsibility of the caller.

Reply:
Sure, refactored.